reportssql/report_util_functions.sql

```diff
@@ -1625,3 +1625,145 @@ BEGIN
     RETURN result;
 END$$
 DELIMITER ;
+
+-- patientRegisteredBeforeReportStartDate
```

Comment:
@tmvumbi2 This functions should be added to the testing_report_util_functions.sql file instead.